src/Umbraco.Web.Common/Extensions/UmbracoApplicationBuilder.Identity.cs

```diff
@@ -0,0 +1,23 @@
+using Microsoft.AspNetCore.Identity;
+using Microsoft.Extensions.DependencyInjection;
+using Microsoft.Extensions.DependencyInjection.Extensions;
+using Umbraco.Cms.Core.DependencyInjection;
+using Umbraco.Cms.Core.Security;
+
+namespace Umbraco.Extensions
+{
+    public static partial class UmbracoApplicationBuilderExtensions
+    {
+        public static IUmbracoBuilder SetBackOfficeUserManager<TUserManager>(this IUmbracoBuilder builder)
+            where TUserManager : UserManager<BackOfficeIdentityUser>, IBackOfficeUserManager
+        {
+
+            var customType = typeof(TUserManager);
+            var userManagerType = typeof(UserManager<BackOfficeIdentityUser>);
```

Comment:
```suggestion
            Type customType = typeof(TUserManager);
            Type userManagerType = typeof(UserManager<BackOfficeIdentityUser>);
```

It's a bit of an obvious one since it comes from `typeof`, but better to keep stylecop happy :)